libp2p/__init__.py

```diff
@@ -105,14 +107,13 @@ def initialize_default_swarm(
         swarm_opt = initialize_default_swarm(
             id_opt=id_opt, transport_opt=transport_opt,
             muxer_opt=muxer_opt, sec_opt=sec_opt,
-            peerstore_opt=peerstore_opt)
+            peerstore_opt=peerstore_opt, disc_opt=disc_opt)
+
+    swarm_opt.add_router(disc_opt)
```

Comment:
I think we can take out this line as disc_opt should have been passed into Swarm in Swarm construction at line 83.